fix-point/src/CopyFiles.hs

```diff
@@ -23,8 +30,9 @@ copyAll source target = do
 
   -- if bottom two lines swapped, infinite creation of target occurs
     content <- listDirectory source
+    let refinedContent = filter (/= (makeRelative homeFP fixPoint)) content
```

Comment:
This can be done simpler. Just `filter (/= "fix-point")`. This is exactly the output of `makeRelative` function. You don't need to store absolute paths as variables in your code.